### PR TITLE
mobile:  Fix mobile navigation, search and logo rendering

### PR DESCRIPTION
**Pull request by dimitrisnl**
Connects-to: #952
Change-type: patch
Signed-off-by: Dimitrios Lytras <dimitrios@balena.io>

### CODE REVIEW

**Comment by dimitrisnl** [comment]
Currently, we parse all the link and apply the `docs` prefix. 
The links in the mobile navigation were not present in the DOM at that time and would not be updated.

Same goes for the image which would be missing the `docs` prefix for its path.

Another side effect was never initializing Algolia.

I chose to remove the `Ractive` library that was handling it. It was only used there so removing this dependency was a good choice.

In my fix, I simply hide the dropdowns and manipulate the height of the elements to make them invisible.

**Comment by lurch** [comment]
Was this supposed to connect to #952 ?

**Comment by dimitrisnl** [comment]
@lurch 	Good catch will update it.

File: static/scripts/main.js [hunk 1]
```diff
@@ -154,24 +153,9 @@ $('.site-navigation')
...
-
-Ractive({
-  el: '#ract-mobile-sub-menu',
-  template: '#tpl-mobile-sub-menu',
-  data: {
-    collapsed: true
-  }
+$('.navbar-toggle').click(function() {
+  $(this).find('span').toggleClass('active')
+  $(this).parent().siblings().toggleClass('collapsed')
```
Comment (thgreasi) [inline static/scripts/main.js]: NIT: Could be
```suggestion
  $(this).closest('navbar').find('navbar-collapse').toggleClass('collapsed')
```

**Pull request merged**